sdks/python/apache_beam/testing/benchmarks/nexmark/README.md

```diff
@@ -23,22 +23,28 @@
 
 For batch mode, a file needs to be generated first by running java suite and writing events to a file.
 
-```shell script
-./gradlew :sdks:java:testing:nexmark:run \
-    -Pnexmark.runner=":runners:direct-java"
-    -Pnexmark.args=" --query=1 --runner=DirectRunner --streaming=false --suite=SMOKE --numEvents=100000  --manageResources=false --monitorJobs=true --enforceEncodability=true --enforceImmutability=true --generateEventFilePathPrefix=YOUR_FILE_PREFIX"
-```
-
 ### Direct Runner
 
 ```shell script
-python nexmark_launcher.py --query 5 --num_events 10000 --runner DirectRunner --input PATH_TO_INPUT_FILE
+./gradlew :sdks:java:testing:nexmark:run \
+    -Pnexmark.runner=":runners:direct-java" \
+    -Pnexmark.args="--query=0 --runner=DirectRunner --numEvents=100000 --manageResources=false --monitorJobs=true --enforceEncodability=true --enforceImmutability=true --generateEventFilePathPrefix=/tmp/eventfile"
+
+./gradlew :sdks:python:apache_beam:testing:benchmarks:nexmark:run \
+    -Pnexmark.args="--query=0 --num_events=100000 --runner=DirectRunner --input=/tmp/eventfile\*"
```

Comment:
It would be interesting to make the former command a pre-requisite for the latter. But I guess that doesn't make sense in the distributed runner + cloud storage case.

Reply:
I agree. I didn't see an easy way to do it so I'm leaving it as two commands for now.